web/crux-ui/src/components/products/versions/images/config/common-config-section.tsx

```diff
@@ -71,6 +75,8 @@ const CommonConfigSection = (props: CommonConfigSectionProps) => {
     config: propsConfig,
   } = props
 
+  const storages = useStorageOptions(t)
```

Comment:
Better way to do this?
Maybe cache them somehow?

Reply:
@m8vago